taipy/core/data/data_node.py

```diff
@@ -167,7 +191,10 @@ def last_edit_date(self):
     @last_edit_date.setter  # type: ignore
     @_self_setter(_MANAGER_NAME)
     def last_edit_date(self, val):
+        prev_last_edit_date = self._last_edit_date
         self._last_edit_date = val
+        if prev_last_edit_date is None and self._last_edit_date:
+            SubmittableStatusCache.remove(self.id)
```

Comment:
I believe this does not work.
We don't know if we need to remove the dn from the cache here, since we don't know the value of `_edits_in_progress`.

Reply:
this has been fixed with your proposal from yesterday